package.json

```diff
@@ -28,15 +28,15 @@
 	"scripts": {
 		"build": "tsc",
 		"postbuild": "cpy README.md ./dist && cpy package.json ./dist && cpy LICENSE ./dist",
-		"format": "prettier --write .",
-		"format:check": "prettier --check .",
-		"generate": "npm-run-all \"generate:*\"",
-		"generate:configs": "ts-node tools/generate-configs && npm run format",
-		"generate:rules-list": "npm run build && eslint-doc-generator --path-rule-list \"../README.md\" --path-rule-doc \"../docs/rules/{name}.md\" --url-rule-doc \"docs/rules/{name}.md\" dist/ && npm run format",
-		"lint": "npm-run-all \"lint:*\"",
-		"lint:generate": "npm run generate && git diff --exit-code",
-		"lint:js": "eslint . --max-warnings 0 --ext .js,.ts",
-		"lint:js:fix": "npm run lint:js -- --fix",
+		"prettier-base": "prettier . --ignore-unknown --cache --loglevel warn",
+		"format": "npm run prettier-base -- --write",
+		"format:check": "npm run prettier-base -- --check",
+		"lint": "eslint . --max-warnings 0 --ext .js,.ts",
+		"lint:fix": "npm run lint -- --fix",
+		"rule-doc-generator": "eslint-doc-generator --path-rule-list \"../README.md\" --path-rule-doc \"../docs/rules/{name}.md\" --url-rule-doc \"docs/rules/{name}.md\" dist/",
+		"docs:gen": "npm-run-all \"docs:gen:*\"",
+		"docs:gen:configs": "ts-node tools/generate-configs && npm run format",
+		"docs:gen:rules-list": "npm run build && npm run rule-doc-generator && npm run format",
```

Comment:
Can we run `npx sort-package-json` again, like it was?

Reply:
I didn't know it was sorted following that package. Sure, I'll run it.

Reply:
I'm running it, but it makes no difference 🤔

Reply:
This is probably because of the `npm-run-all` call
By default it's running these in sequences, it should use the parallel flag instead.

```json
"docs:gen": "run-p \"docs:gen:*\"",
```

Same for `generate` & `lint` script

Reply:
I'm completely lost now. How is this related to `npx sort-package-json`?

Reply:
If `npm-run-all` is running in sequence, the order of the scripts matter
`sort-package-json` is taking this into account.

For more info see https://github.com/keithamus/sort-package-json/issues/220 & https://github.com/keithamus/sort-package-json/pull/232

Temporary solution would be to remove `npm-run-all` from `devDependencies`, run `npx sort-package-json` & add `npm-run-all` to `devDependencies` again

Reply:
Not sure if I understood this correctly at all.

I was just running `sort-package-json` manually to sort the package.json according to it. But it seems you are referring to running this in one script as before? I don't see where it was executed prior to my changes.

Reply:
You indeed need to run it manually, but first need to (temporarily) remove the `npm-run-all` dependency, otherwise it won't change anything

Reply:
Ok, I don't understand any of this, but I finally got it 😂